commons/push/integration/garmin/garmin_push_activity_details.avsc

```diff
@@ -0,0 +1,37 @@
+  "fields": [
+    {"name": "summaryId", "type": "string", "doc": "Unique identifier for the summary."},
+    {"name": "time", "type": "double", "doc": "Start time of the activity in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "timeReceived", "type": "double", "doc": "Time that this record was collected by a service in seconds since the Unix Epoch (s)."},
+    {"name": "startTimeOffsetInSeconds", "type": ["null", "int"], "doc": "Offset in seconds to add to time to derive the 'local' time of the device that captured the data.", "default": null},
+    {"name": "activityType", "type": ["null", "string"], "doc": "Text description of the activity type. See Appendix A for a complete list.", "default": null},
+    {"name": "durationInSeconds", "type": ["null", "int"], "doc": "Length of the monitoring period in seconds.", "default": null},
+    {"name": "averageBikeCadenceInRoundsPerMinute", "type": ["null", "double"], "doc": "Average Bike Cadence in rounds per minute.", "default": null},
+    {"name": "maxBikeCadenceInRoundsPerMinute", "type": ["null", "double"], "doc": "Maximum Bike Cadence in rounds per minute.", "default": null},
+    {"name": "distanceInMeters", "type": ["null", "double"], "doc": "Distance traveled in meters.", "default": null},
+    {"name": "averageRunCadenceInStepsPerMinute", "type": ["null", "double"], "doc": "Average Run Cadence in Steps per minute.", "default": null},
+    {"name": "maxRunCadenceInStepsPerMinute", "type": ["null", "double"], "doc": "Maximum Run Cadence in Steps per minute.", "default": null},
+    {"name": "activeKilocalories", "type": ["null", "int"], "doc": "Active kilocalories (dietary calories) burned through actual movement and activity during the monitoring period.", "default": null},
+    {"name": "averageSpeedInMetersPerSecond", "type": ["null", "double"], "doc": "Average Speed in meters per second.", "default": null},
+    {"name": "maxSpeedInMetersPerSecond", "type": ["null", "double"], "doc": "Maximum Speed in meters per second.", "default": null},
+    {"name": "averageSwimCadenceInStrokesPerMinute", "type": ["null", "double"], "doc": "Average Swim Cadence in strokes per minute.", "default": null},
+    {"name": "averagePaceInMinutesPerKilometer", "type": ["null", "double"], "doc": "Average Pace in minutes per kilometer.", "default": null},
+    {"name": "maxPaceInMinutesPerKilometer", "type": ["null", "double"], "doc": "Maximum Pace in minutes per kilometer.", "default": null},
```

Comment:
All of these values and the following do not have enough digits to warrant double precision, float should be sufficient: [it can store 6 digits exactly](https://www.exploringbinary.com/decimal-precision-of-binary-floating-point-numbers/). Please make this evaluation for other records as well.